app/models/appointment.rb

```diff
@@ -1,4 +1,12 @@
 class Appointment < ApplicationRecord
   belongs_to :user
   belongs_to :doctor
+
+  # validations
+  # validate :date_of_appointment_cannot_be_in_the_past
+  validates :description, presence: true, length: { in: 3..150 }
+  validates :date_of_appointment, presence: true,
```

Comment:
[OPTIONAL]Great job, I like the fact that the appointment has to be in the future, however, I would suggest removing the date format (Rails will convert the date to meet the format used) requirements as it simplifies the user experience.